### DISCUSSION TITLE
Help Please

### DISCUSSION THREAD

**Discussion by lloydwoods** [lmusarella/Pancakeswap-Prediction-Bot-2023#29] [Q&A]
@lmusarella I've got someone running a sweeper bot on a wallet that has a LOT of locked Cake in it and I had my expiration date wrong.  If there's any way you'd be willing to help (for pay of course) please reach out to me on X @lloydwoods (follow first) or the same handle @g mail.

**Comment by lloydwoods** [comment]
> @lmusarella I've got someone running a sweeper bot on a wallet that has a LOT of locked Cake in it and I had my expiration date wrong. If there's any way you'd be willing to help (for pay of course) please reach out to me on X @lloydwoods (follow first) or the same handle @g mail.

I feel like you'd be able to help me modify this script to do the necessary tasks but I can't quite figure it out myself.

**Reply by lmusarella** [reply]
Hi, sorry, I don't understand well, what problem do you have? Is this a bot-related issue?

**Reply by kabiruharuna** [reply]
Hello i have affected wallets with similar problem, is there a way you can help?

**Reply by jtruji68** [reply]
I think he's asking for help with another problem, and see if you can help him with that, I don't think its related to the bot at all. Maybe a DM would be more appropriate.

**Comment by lmusarella** [comment]
I Need more information pls

Il gio 19 ott 2023, 22:53 Kabiru Haruna ***@***.***> ha
scritto:
